engine/include/cubos/engine/physics/components/force.hpp

```diff
@@ -0,0 +1,39 @@
+/// @file
+/// @brief Component @ref cubos::engine::Force.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <glm/vec3.hpp>
+
+#include <cubos/core/reflection/reflect.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which holds forces applied on a particle.
+    /// @note Should be used with @ref Velocity.
+    /// @ingroup physics-plugin
+    struct Force
+    {
+        CUBOS_REFLECT;
+
+        glm::vec3 getForce() const
+        {
+            return mForce;
+        }
+
+        void addForce(glm::vec3 force)
+        {
+            mForce += force;
+        }
+
+        void clearForce()
+        {
+            mForce = {0.0F, 0.0F, 0.0F};
+        }
+
+    private:
+        glm::vec3 mForce = {0.0F, 0.0F, 0.0F};
```

Comment:
I like the field public for when people are just accessing the value but I don't like that people will probably use it instead of the method to add things, bc the method is only for consistency when we get into rigid bodies. We can make it public and keep the method and use it on internal stuff and then the user does whatever they want in their code maybe? 
As for the names, I think `vec` is a good choice and I agree with removing the suffixes.

Reply:
Yeah you've convinced me. In that case though, I would suggest renaming the getter to `vec()`, so that it can be used as `force.vec()`.

Reply:
Ok, the other day I forgot something. The reason the methods have the suffix `Force` is because the idea would be to have torque in the force component since linear and angular force are both forces (and the equivalent for the impulses). So, in the end we'd have like 3 methods `addForce`, `addTorque` and `addForceOnPoint` to add, for example, and that's why we needed the suffix. Adding force on a point will mess with linear and angular force, so that's why they are both in the same component, I'm not sure how we could separate them. Do you have any suggestions to this idea?